messaging/src/main/java/de/fraunhofer/ids/messaging/protocol/MessageService.java

```diff
@@ -73,9 +76,11 @@ public MessageService(final IdsHttpService httpService) {
             throws
             MultipartParseException,
             ClaimsException,
-            IOException,
             UnknownResponseException,
-            DeserializeException {
+            DeserializeException,
+            SerializeException,
+            IOException,
```

Comment:
Is this still thrown?

Reply:
yes, due to the sendAndCheckDat method, which internally calls OkHttpClient.call -> IOException can be thrown if target not reachable